lld/ELF/Arch/X86_64.cpp

```diff
@@ -932,7 +937,18 @@ static void relaxGotNoPic(uint8_t *loc, uint64_t val, uint8_t op,
     // REX.B This 1-bit value is an extension to the MODRM.rm field or the
     // SIB.base field.
     // See "2.2.1.2 More on REX Prefix Fields " (2-8 Vol. 2A).
-    loc[-3] = (rex & ~0x4) | (rex & 0x4) >> 2;
+    //
+    // REX2 prefix is encoded as 0xd5|M|R2|X2|B2|WRXB, where
+    // 0xd5 is 1byte fixed pattern.
+    // REX2's [W,R,X,B] have the same meanings as REX's.
+    // REX2.M encodes the map id.
+    // R2/X2/B2 provides the fifth and most siginicant bits of the R/X/B
+    // register identifiers, each of which can now address all 32 GPRs.
+    // TODO: Add the section number here after APX SPEC is merged into SDM.
```

Comment:
Is this TODO useful?

Reply:
I see `See "2.2.1.2 More on REX Prefix Fields " (2-8 Vol. 2A).` at line 939. 

Currently APX stuff is in a separate SPEC https://cdrdv2.intel.com/v1/dl/getContent/784266.

I can drop this if you think it's not needed.

Reply:
This seems too specific and may not be needed :)

Reply:
Done